data/transactions/logic/sourcemap.go

```diff
@@ -49,24 +51,35 @@ func GetSourceMap(sourceNames []string, offsetToLine map[int]int) SourceMap {
 		}
 	}
 
-	// Array where index is the PC and value is the line.
+	// Backwards compatibility for `mapping` field.
+	oldPcToLine := make([]string, maxPC+1)
+	for pc := range oldPcToLine {
+		if line, ok := offsetToLine[pc]; ok {
+			oldPcToLine[pc] = MakeSourceMapLine(0, 0, line, 0)
```

Comment:
@algochoi Apologies if I created confusion here - Do you agree with the thinking below?

I imagined `mapping` will be populated with a relative value rather than the absolute value.  Since the absolute line number does _not_ abide by the spec, I think we ought to fix the behavior in all places.

Reply:
Had a quick with @algochoi and agreed to change to _relative_ value for `mapping` rather than preserving the as was (absolute) value.

Rationale:  The prior implementation deviates from a specification.  Downstream users expect specification conformance.  Consequently, leaving as is (absolute value) precludes downstream users from using sourcemap tooling.

Leaving thread open a visual reminder until PR updates happen.